SU2_CFD/src/python_wrapper_structure.cpp

```diff
@@ -136,3 +136,29 @@ void CDriver::BoundaryConditionsUpdate() {
     geometry_container[iZone][INST_0][MESH_0]->UpdateCustomBoundaryConditions(geometry_container[iZone][INST_0],config_container[iZone]);
   }
 }
+
+////////////////////////////////////////////////////////////////////////////////
+/* Functions related to dynamic mesh */
+////////////////////////////////////////////////////////////////////////////////
+
+void CDriver::SetTranslationRate(passivedouble xDot, passivedouble yDot, passivedouble zDot) {
```

Comment:
You can keep them here for now, I may give them a new home as the python wrapper work continues.

Reply:
OK :)